src/syft/lib/petlib/ec.py

```diff
@@ -0,0 +1,50 @@
+# stdlib
+
+# third party
+import petlib
+
+# syft relative
+from ...generate_wrapper import GenerateWrapper
+from ...proto.lib.petlib.ec_obj_pb2 import EC2_obj
+
+
+def object2proto(obj: object) -> EC2_obj:
+    proto = EC2_obj()
+    proto.group_nid = obj.group.nid()  # type: ignore
+    proto.vector = obj.export()  # type: ignore
+    return proto
+
+
+def proto2object(proto: EC2_obj) -> petlib.ec.EcPt:
+
+    vec = petlib.ec.EcPt(proto.vector, petlib.ec.EcGroup(proto.group_nid))
+    return vec
+
+
+GenerateWrapper(
+    wrapped_type=petlib.ec.EcPt,
+    import_path="petlib.ec.EcPt",
+    protobuf_scheme=EC2_obj,
+    type_object2proto=object2proto,
+    type_proto2object=proto2object,
+)
+
+
+def object2proto_group(obj: object) -> EC2_obj:
+    proto = EC2_obj()
+    proto.group_nid = obj.group.nid()  # type: ignore
+    return proto
+
+
+def proto2object_group(proto: EC2_obj) -> petlib.ec.EcGroup:
```

Comment:
I don't think these should share a protobuf, maybe just split it out to a seperate protobuf.